src/services/job-service.ts

```diff
@@ -12,7 +14,21 @@ export class JobService {
 
     public start(): void {
         for (let job of this.jobs) {
-            schedule.scheduleJob(job.schedule, async () => {
+            let jobSchedule: Date | string | RecurrenceSpecDateRange = job.runOnce
```

Comment:
There's a bunch of possible types here:
![image](https://github.com/KevinNovak/Discord-Bot-TypeScript-Template/assets/11231402/ae5339c2-59e9-4ce7-a70b-7571edb94628)

Can we just leave it at Date | string for simplicity?

Reply:
Using ReccurenceSpecDateRange allows initialDelaySeconds to work independently from runOnce. We use it specify a schedule but also a start date.

Reply:
Can we remove the types then and let it auto-detect? Just `let jobSchedule = ...`

Reply:
I guess, it's not necessary xD